experiments/main_agentic/plan.py

```
@@ -0,0 +1,1232 @@
+        f"Split: {plan.effective_split}",
+        "Selection mode: "
+        + (
+            "force all selected runs"
+            if preview.selection.no_skip_completed
+            else (
+                "rerun matching statuses: " + ", ".join(preview.selection.rerun_statuses)
+                if preview.selection.rerun_statuses
+                else "default artifact-first resume"
+            )
+        ),
+        f"Cases per benchmark: "
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)